ext/include/opentelemetry/ext/http/client/curl/http_operation_curl.h

```diff
@@ -300,7 +301,7 @@ class HttpOperation
   const opentelemetry::ext::http::client::Compression &compression_;
 
   // Processed response headers and body
-  long response_code_;
+  int64_t response_code_;
```

Comment:
This relates to CURLINFO_RESPONSE_CODE, which uses a long.

Revert.

Reply:
Noted and reverted.